### ISSUE TITLE
Startup Profit Prediction: Model Training using Linear Regression

### ISSUE THREAD

**Issue by ISaumya1011** [GDG-IGDTUW/AI-ML-1#4]
Training on Train set - Use Linear Regression

**Comment by ayushibhati** [comment]
Please assign this issue to me @ISaumya1011 @Anamika-Thakur

**Comment by Saisha0512** [comment]
Hi Di, Please assign this issue to me.
Maintainer ID : @ISaumya1011
Collaborator ID : @Anamika-Thakur

**Comment by Khwaaish** [comment]
@ISaumya1011 and @Anamika-Thakur please assign this issue to me

**Comment by Anamika-Thakur** [comment]
Issue has been assigned to @ayushibhati @Khwaaish @Saisha0512 . Whoever completes it first their PR will be merged. Also make sure there are no changes except for the cell that is concerned with your issue. Thank you. All the best!

**Comment by Saisha0512** [comment]
Hi Di @Anamika-Thakur , I can't run my code because the code required for ISSUE - 2 is not there in the cell. Kindly look into it. Should I submit it without running the file?

**Comment by Anamika-Thakur** [comment]
Yes you can try it once it is solved by the one who has been assigned the issue.
